loom/script/native/core/system/lmConsole.cpp

```diff
@@ -36,8 +36,9 @@ int Console::print(lua_State *L)
     lua_rawgeti(L, 1, LSINDEXVECTOR);
     int vidx = lua_gettop(L);
 
-    // the value to print (concating all objects passed)
-    utString toprint;
+    // the value to print (concatenating all objects passed)
+    char outputBuffer[2048];
```

Comment:
Maybe we should just transition to using the stack buffer if possible (total length being < 2048 or whatever) and allocating if there's too much to print. Perhaps for a different PR.